src/value-converters/abstract-locale-value-converter.ts

```diff
@@ -33,13 +33,13 @@ export abstract class AbstractLocaleValueConverter {
      * Called when the instance or system locale has changed
      */
     protected localeChanged(locale:string) {
```

Comment:
It's not clear to me what the purpose of this function is. 
My interpretation: An "optionally overridable" function to be used in inheritors of this class.
=> To make the compiler error go away, the parameter needs to do something, so there it is.

Reply:
Done.